pkg/sql/testdata/pg_catalog_tables_from_postgres.json

```diff
@@ -1,5 +1,12 @@
 {
-  "pgVersion": "13.2",
+  "pgVersion": "13.3",
+  "diffSummary": {
```

Comment:
why did a diffSummary get added to this file? isn't this file just supposed to be a summary of the tables from postgres?

Reply:
Initially when we did the diff tool we shared the structures to create these files (Both tables and diffs uses same datastructures) I can change this, but I suggest to use a different diff as it will require a big refactor. 

Another option I was thinking that we should merge both files and make them "more readable" so instead of mantaining 2 files (tables and diffs) we would maintain 1.